easybuild/easyconfigs/l/longread_umi/longread_umi-0.3.2-foss-2020b.eb

```diff
@@ -0,0 +1,80 @@
+easyblock = 'Bundle'
+
+name = 'longread_umi'
+version = '0.3.2'
+
+homepage = 'https://github.com/SorenKarst/longread_umi'
+description = "A collection of scripts for processing longread UMI data."
+github_account = 'SorenKarst'
+
+toolchain = {'name': 'foss', 'version': '2020b'}
+
+dependencies = [
+    ('Python', '3.8.6'),
+    ('cutadapt', '3.4'),
+    ('BCFtools', '1.11'),
+    ('BWA', '0.7.17'),
+    ('Filtlong', '0.2.0'),
+    ('gawk', '5.1.0'),
+    ('medaka', '1.4.3'),
+    ('minimap2', '2.18'),
+    ('parallel', '20210322'),
+    ('Pysam', '0.16.0.1'),
+    ('Racon', '1.4.21'),
+    ('SAMtools', '1.11'),
+    ('seqtk', '1.3'),
+    ('VSEARCH', '2.18.0'),
+]
+
+components = [
+    (name, version, {
+        'easyblock': 'Tarball',
+        'source_urls': [GITHUB_LOWER_SOURCE],
+        'sources': ['v%(version)s.tar.gz'],
```

Comment:
@zarybnicky Since we're download sources for two different components, it's worth renaming the source tarballs on download:

```python
        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}],
```

Likewise for `Porechop`, and I would move `Porechop` as first component (since it's a dep for `longread_umi`), although the installation order doesn't actually matter.